x/profiles/keeper/grpc_query.go

```diff
@@ -103,8 +103,30 @@ func (k Keeper) UserRelationships(ctx context.Context, request *types.QueryUserR
 // UserBlocks implements the Query/UserBlocks gRPC method
 func (k Keeper) UserBlocks(ctx context.Context, request *types.QueryUserBlocksRequest) (*types.QueryUserBlocksResponse, error) {
```

Comment:
Can you add a test for this please? Doing so would probably fix the coverage issues